lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -426,7 +428,9 @@ def main():
             choices=['present', 'absent', 'maintenance', 'unattached'],
             default='present',
         ),
+        id=dict(default=None),
         name=dict(required=True),
+        import_=dict(type='bool', default=False),
```

Comment:
sure, I will try to indicate a new state for that
all storage domains can be imported basically, but it works differently for NFS and block.
in NFS if the storage exists then the engine assumes we mean to import it, in block we need to specifically indicate the flag